packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/async_jobs/async_jobs.py

```diff
@@ -103,3 +121,111 @@ async def submit(
     )
     assert isinstance(_result, AsyncJobGet)  # nosec
     return _result
+
+
+_DEFAULT_RPC_RETRY_POLICY: dict[str, Any] = {
+    "retry": retry_if_exception_type(RemoteMethodNotRegisteredError),
+    "wait": wait_random_exponential(max=20),
+    "stop": stop_after_delay(60),
+    "reraise": True,
+    "before_sleep": before_sleep_log(_logger, logging.INFO),
+}
+
+
+@retry(**_DEFAULT_RPC_RETRY_POLICY)
+async def _wait_for_completion(
+    rabbitmq_rpc_client: RabbitMQRPCClient,
+    *,
+    rpc_namespace: RPCNamespace,
+    job_id: AsyncJobId,
+    job_id_data: AsyncJobNameData,
+    client_timeout: int,
```

Comment:
below this is a `client_timeout: datetime.timedelta,`

Reply:
👍